common.gypi

```diff
@@ -269,7 +269,8 @@
           ['clang==1', {
             'xcode_settings': {
               'GCC_VERSION': 'com.apple.compilers.llvm.clang.1_0',
-              'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++0x',  # -std=gnu++0x
+              'CLANG_CXX_LANGUAGE_STANDARD': 'c++11',  # -std=c++11
+              'CLANG_CXX_LIBRARY': 'libc++', #-stdlib=libc++
```

Comment:
ok, we just need an eye on portability - can't have people using the linux builds and then have them complaining about missing symbols because they don't have stdc++ junk installed